js/node/test/test-utils.ts

```diff
@@ -274,3 +277,19 @@ export function shouldSkipModel(model: string, eps: string[]): boolean {
 
   return false;
 }
+
+const OVERRIDES: {[key: string]: (number|{[name: string]: number})} =
```

Comment:
as "atol_default" and "rtol_default" are special keys in override json, you can directly define them as in the type of OVERRIDES. so the `as` keyword can be removed in the following few lines.

Reply:
also, could you move this to the top next to BACKEND_TEST_SERIES_FILTERS?

Reply:
Done.